openapi/openapi.yaml

```diff
@@ -604,6 +604,199 @@ paths:
               schema:
                 $ref: "#/components/schemas/AnalysisResponseError"
 
+  /hardware-envrionment:
+    get:
+      tags: [Info]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: list_hardware_environments
+      summary: Retrieve a list of supported hardware environments
+      parameters:
+        - name: is_external
+          in: query
+          required: false
+          description: >
+            Get external hardware environments.
+          schema:
+            type: boolean
+            default: false
+        - name: page
+          in: query
+          required: false
+          description: Page offset in pagination.
+          schema:
+            type: integer
+            default: 0
+      responses:
+        "200":
+          description: A list of supported hardware environments.
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - parameters
+                  - hardware_environments
+                properties:
+                  parameters:
+                    type: object
+                  hardware_environments:
+                    type: array
+                    items:
```

Comment:
@fridex maybe I should get rid of this long list of items structure, WDYT?

Reply:
Please keep it. It's used during automatic generation of code in the clinet - thamos (and it's good to see what is returned from the API).